angular/_margins.scss

```diff
@@ -36,5 +37,142 @@
   }
 
   @mixin bodyfont(){
-    font-family: "Open Sans", Arial, Helvetica, sans-serif;
-  }
\ No newline at end of file
+    font-family: "Open Sans", Arial, Helvetica, sans-serif; 
+  }
+  @mixin darktheme(){
```

Comment:
move darktheme mixin to another file: _darkTheme.scss

Reply:
configure the mixin to take a primary & accent parameter, so we can reuse it for future themes with different colors